docs/markdown/Using-wraptool.md

```diff
@@ -64,7 +64,12 @@ the install command.
     Installed libjpeg branch 9a revision 2
 
 Now you can issue a `subproject('libjpeg')` in your `meson.build` file
-to use it.
+to use it. Since *1.5.0* you may also specify multiple dependencies to install.
```

Comment:
Since this missed the previous release(s), through no fault of your own, this version number does need updating again. Sorry. :(